.github/workflows/JetNews.yaml

```diff
@@ -45,6 +45,10 @@ jobs:
         working-directory: ${{ env.SAMPLE_PATH }}
         run: ./gradlew spotlessCheck assembleDebug lintDebug --stacktrace
 
+      - name: Run local tests
```

Comment:
Run the `testDebug` task in the `gradlew` task above? That lets Gradle schedule it as appropriate.

Reply:
I don't know what that means but sure